coc.md

```diff
@@ -0,0 +1,106 @@
+# Women Who Go - Code of Conduct
+
+GoBridge (GB) and Women Who Go (WWG) uses the Go CoC as our base document:
+
+[Go CoC](https://golang.org/conduct) | [Go Forum](https://forum.golangbridge.org/) | [Gophers Slack](http://invite.slack.golangbridge.org) | [Women Who Go Slack](https://womenwhogo.org/invite.html)
```

Comment:
Fixes a typo s/uses/use and links to the Go CoC when we mention it in the first sentence:

```suggestion
GoBridge (GB) and Women Who Go (WWG) use the [Go CoC] as our base document:

[Go Forum](https://forum.golangbridge.org/) | [Gophers Slack](http://invite.slack.golangbridge.org) | [Women Who Go Slack](https://womenwhogo.org/invite.html)

[Go CoC]: https://golang.org/conduct
```

It is confusing if our CoC applies to the gobridge forum and slack since they are listed here. I suggest limiting this document to just WWG moderated spaces.